CONTRIBUTING.md

```diff
@@ -28,3 +28,51 @@ Members of the Contributors team are encouraged to review pull requests that
 have already been reviewed, and pull request contributors are encouraged to seek
 multiple reviews.  Reviews from anyone not on the Contributors team are always
 appreciated and encouraged!
+
+## Reviewing Pull Requests
+
+All mlpack contributors who choose to review and provide feedback on Pull
+Requests have a responsibility to both the project and the individual making
+the contribution. 
+
+Reviews and feedback must be
```

Comment:
This is a really minor language issue, but I think we should go with `should be` instead of `must be` here.  The reason I say that is that the document here should seem friendly and inviting (and we should as a community too of course :smile:), so we wouldn't want to write anything that makes people think "wow, reviewing pull requests for mlpack is serious business, I better be careful because I don't want to break the rules".  Although we definitely should be striving to make all reviews and feedback friendly and welcoming, I think `should be` is sufficient to get us all pointed towards that goal. :)

Reply:
yeah correct